serveradmin/nessus/templates/nessus/nessus.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+
+{% block title %}
+    Scan configuration
+{% endblock %}
+
+{% block content %}
+<div class="row">
+    <div class="col-md-1"></div>
+    <div class="col-md-9">
+        <h3>Scan configuration</h3>
+    </div>
+</div>
+<div class="row">
+    <div class="col-md-1"></div>
+    <div class="controls col-md-9">
+        <form>
+            <p>
+            <div class="form-group row input-controls">
+                <label for="host" class="col-sm-1 col-form-label">Hosts:</label>
+                <div class="col-sm-8">
+                    <input id="hosts" name="hosts" type="text" value="{{ hostnames|join:", " }}" class="form-control form-control-sm" />
+                </div>
+            </div>
+            <div class="form-group row input-controls">
+                <label for="type" class="col-sm-1 col-form-label">Scan Type:</label>
+                <div class="col-sm-8">
+                    <select id="type" name="type" class="form-select form-control form-control-sm">
+                        <option>network</option>
+                        <option>web</option>
+                    </select>
+                </div>
+            </div>
+            <div class="form-group row input-controls">
+                <label for="email" class="col-sm-1 col-form-label">Receiving email:</label>
+                <div class="col-sm-8">
+                    <input id="email" name="email" type="text" value="user@email.com" class="form-control form-control-sm" />
```

Comment:
You could pre-fill this with the current users E-Mail address

In the views you can access the user object from the request. Then you can pass it as variable to the template.
```
email = request.user.email
```